src/lib.rs

```diff
@@ -341,10 +341,6 @@
 #![warn(missing_docs)]
 #![deny(missing_debug_implementations, unconditional_recursion)]
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
-// When compiled for the rustc compiler itself we want to make sure that this is
-// an unstable crate
-#![cfg_attr(rustbuild, feature(staged_api, rustc_private))]
```

Comment:
I believe this is a left-over. As far as I know `log` isn't used in the compiler anymore, but will actually go and check.

Reply:
Looks like it's used in some tools: https://github.com/search?q=log+repo%3Arust-lang%2Frust+path%3A*.toml&type=code&ref=advsearch

Looking at other crates.io dependencies, they don't have any special handling to appear in the compiler anymore so I think this is still safe to remove.